automation/repeated_test.sh

```diff
@@ -158,17 +178,31 @@ if should_skip_test_run_due_to_too_many_tests "${NEW_TESTS}"; then
     exit 0
 fi
 
+# for migration tests we need three nodes, thus we need to check whether migration tests are going to be run
+KUBEVIRT_NUM_NODES=2
+if [[ "${NEW_TESTS}" =~ migration ]]; then
+    KUBEVIRT_NUM_NODES=3
+else
+    if grep -q 'SkipTestIfNotEnoughNodesWithCPUManager' $(echo ${NEW_TESTS} | tr '|' ' '); then
```

Comment:
Does `$NEW_TESTS` include the entire code of each test? That skip is a function that's called by a test, it's not part of the test entry line like a decorator would.
I'd suggest creating a new decorator to be able to easily and reliably identify the (currently 3) tests that need 3 nodes.

Reply:
No, `NEW_TESTS` only contains the file names of the touched test files.